app/views/RoomsListView/Header/Header.tsx

```diff
@@ -62,9 +64,16 @@ const Header = React.memo(
 		const { status: supportedVersionsStatus } = useAppSelector(state => state.supportedVersions);
 		const { colors } = useTheme();
 		const { width: windowWidth } = useWindowDimensions();
+		const fontScale = PixelRatio.getFontScale();
 
 		if (showSearchHeader) {
-			return <SearchHeader onSearchChangeText={onSearchChangeText} testID='rooms-list-view-search-input' />;
+			return (
+				<SearchHeader
+					onSearchChangeText={onSearchChangeText}
+					testID='rooms-list-view-search-input'
+					style={{ height: 36.9 * fontScale }}
```

Comment:
Too magical? I'm not sure a very precise number like that would make any difference on some Android devices depending on their pixel density